k8s-install/README.md

```diff
@@ -1,11 +1,18 @@
 # Kubernetes self-hosted install
 
-Installation is simple:
+Installation is simple, for Kubernetes 1.5:
 
 ```
 kubectl apply -f canal.yaml
 ```
 
+For Kubernetes 1.6:
+
+```
+kubectl apply -f canal.yaml
+kubectl apply -f rbac.yaml
```

Comment:
@caseydavenport @djosborne Should I make this a URL similar to how the kubeadm section has it?